shared/rspec/rails_helper.rb

```diff
@@ -11,6 +11,7 @@
 ActiveRecord::Migration.maintain_test_schema!
 
 RSpec.configure do |config|
+  # rubocop:disable Lint/InterpolationCheck
   config.fixture_path = "\#{::Rails.root}/spec/fabricators"
```

Comment:
Why not fixing it with:

```
File.join(::Rails.root, 'spec', 'fabricators')
```

Reply:
@olivierobert I think this one `Rails.root.join('spec', 'fabricators')` is enough